pkg/processor/sumologicschemaprocessor/processor.go

```diff
@@ -53,7 +69,9 @@ func newSumologicSchemaProcessor(set component.ProcessorCreateSettings, config *
 func (processor *sumologicSchemaProcessor) start(_ context.Context, host component.Host) error {
 	processor.logger.Info(
 		"Processor sumologic_schema has started.",
-		zap.Bool("add_cloud_namespace", processor.cloudNamespaceProcessor.addCloudNamespace),
+		zap.Bool("add_cloud_namespace", processor.subprocessors[0].isEnabled()),
```

Comment:
Please move `"add_cloud_namespace"` and so on into subprocessor, as now we have to remember processors order

Reply:
or maybe let's give them name?

Reply:
Adding these properties would be a pain, because we would have to edit every subprocessor while adding a new one with new configuration. Another idea is to make another method, for example `config_name()` which would return an adequate name (`add_cloud_namespace` for the cloudNamespaceProcessor and so on).

Reply:
> because we would have to edit every subprocessor while adding a new one with new configuration

why? we could use property the same way like mentioned `config_name()`

Reply:
Ok, maybe I didn't understand well. I thought you meant adding fields like `add_cloud_namespace` to every subprocessor. Could you explain your idea in more detail?

Reply:
Oh, I think I get it now. Sorry for misunderstanding.

Reply:
I mean more less:
```
type SubProcess struct {
  name string
}

type translateTelegrafMetricsProcessor struct {
        SubProcess
	shouldTranslate bool
}

```